.github/workflows/unittests.yml

```diff
@@ -0,0 +1,39 @@
+ on:
+   push:
+     branches:
+     - '*'
+   pull_request:
+     branches:
+     - '*'
+
+ jobs:
+   unittests:
+     name: conda (${{ matrix.os }}, ${{ matrix.environment-file }})
+     runs-on: ${{ matrix.os }}
+     timeout-minutes: 30 
+     strategy:
+       matrix:
+         os: ['macos-latest', 'ubuntu-latest', 'windows-latest']
+         environment-file: [ci/36.yaml, ci/37.yaml]
+     steps:
+       - uses: actions/checkout@v2
+       - uses: goanpeca/setup-miniconda@v1
+         with:
+            miniconda-version: 'latest'
+            auto-update-conda: true
+            auto-activate-base: false
+            environment-file: ${{ matrix.environment-file }}
+            activate-environment: test
+       - shell: bash -l {0}
+         run: conda info --all
+       - shell: bash -l {0}
+         run: conda list
+       - shell: bash -l {0}
+         run: conda config --show-sources
+       - shell: bash -l {0}
+         run: conda config --show
+       - shell: bash -l {0}
+         run: pip install -e . --no-deps --force-reinstall
```

Comment:
we do not need this command. using `actions/checkout@v2` and `goanpeca/setup-miniconda@v1` should take care of installing all dependencies. i'll make new PR to fix this.

Reply:
without 36, when does `spopt` get installed?

Reply:
This is purpose of `actions/checkout@v2`. It checkout out the `spopt` repo.

Reply:
i see. it just doesnt bother to actually install the module since you're guaranteed to be in the directory